content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,133 @@
+sprawling out far and wide. As adoption grew, [contributors knew there was a need for standardization](https://github.com/kubernetes/community/issues/2466).
+
+This new attention to how the community communicates led to a discovery
+of a complex web of options. There were so many options, it was a
+challenge for anyone to be sure they were in the right place to receive
+the right information. We started taking immediate action by pruning the
+inactive mailing lists and other platforms, combining where we could,
+and thinking about the future. We also asked for feedback from all our
+contributors directly [**via annual surveys**](https://github.com/kubernetes/community/tree/master/sig-contributor-experience/surveys)
+to see where folks were actually reading the news that influences their
+experiences here in our community.
+
+![Kubernetes channel access](https://user-images.githubusercontent.com/1744971/79478603-3a3a1980-7fd1-11ea-8b7a-d36aac7a097b.png)
+
+Our contributor base is larger than many enterprise companies. You can
+only imagine what it’s like getting important messages across to make
+sure they are landing and folks are taking action. 
+
+## Contributing to better communication
+
+Think about how your company/employer solves for this. Many have done so
+by building internal marketing and communication focus areas in
+marketing departments. So that’s what we are doing. This has also been
+applied [at Fedora](https://fedoraproject.org/wiki/Marketing) and at a smaller
+scale in our very [own release](https://github.com/kubernetes/sig-release/tree/master/release-team) and [contributor summit](https://github.com/kubernetes/community/blob/d0fd6c16f7ee754b08082cc15658eb8db7afeaf8/events/events-team/marketing/README.md) planning
+teams as roles. 
+
+We have hit the accelerator on an **upstream marketing group** under SIG
+Contributor Experience, and we want to tackle this challenge straight
+on. We’ve learned in other contributor areas that creating roles for
+contributors is super helpful - onboarding, breaking down work, and
+ownership. [Here’s our team charting the course](https://github.com/kubernetes/community/tree/master/communication/marketing-team).
+
+Journey your way through our other documents like our charter if you are
+interested in our mission and scope.
+
+Many of you close to the ecosystem might be scratching your head - isn’t
+this what CNCF does?
+
+Yes and no. The CNCF has 40+ other projects that need to be marketed to
+a countless number of different types of community members in distinct
+ways and they aren’t responsible for the day to day operations of their
+projects. They absolutely do partner with us to highlight what we need
+and when we need it, and they do a fantastic job of it (one example is
+the [*@kubernetesio twitter account*](https://twitter.com/kubernetesio?lang=en) and its 200,000
+followers). 
+
+Where this working group differs is in its scope: we are entirely
```

Comment:
nix the working in "working group" here as it sounds too close to our [full fledged working groups](https://github.com/kubernetes/community/blob/master/sig-list.md#master-working-group-list) after reading. Ours is a part of a subproject of contribex (which sounds long and complex for this - you tell me)

Reply:
Maybe just subproject? or group? "Where this group differs"

Reply:
I would say subproject, under ContribEx

Reply:
i went with group because the sentiment is more group than subproject of contribex etc